src/pages/Signup/Signup.js

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import styles from './Signup.module.css';
+import Button from 'react-bootstrap/Button';
+
+
+function SignUpPage() {
+    return (
+        <div className={styles.container}>
+            <div className={styles.signup_title}>
+                <h1>Sign Up for Bear Buddies</h1>
+            </div>
+            <div>
+                <h4>Create a free account or <a className={styles.blue_txt} href="../Login/login" style={{textDecoration: 'none', font: 'inherit'}}>login</a></h4>
+            </div>
+            <div className={styles.data_container}>
+                <form>
+                    <div className={styles.col_cont}>
+                        <div className={styles.col_items}>
+                            <label>first name</label>
+                            <input type="text" className="form-control"></input>
+                        </div>
+                        <div className={styles.col_items}>
+                            <label>last name</label>
+                            <input type="text" className="form-control"></input>
+                        </div>
+                    </div>
+                    <div className={styles.long_text_box}>
+                        <label>email</label>
+                        <input type="text" className="form-control"></input>
+                    </div>
+                    <div className={styles.col_cont}>
+                        <div className={styles.col_items}>
+                            <label>gender</label>
+                            <select className="form-control">
+                                <option value="" disabled selected hidden>select gender&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;⌄</option>
```

Comment:
LOL, this is really jank. Instead, add an SVG logo and align it to the right of the input box.